README.md

```diff
@@ -1,70 +1,114 @@
-# Getting Started with Create React App
+# math-magician 
+Just a simple webpack template for my projects
```

Comment:
- [ ] I suggest you remove this heading section since it refers to a different project. 🤓 🔧